### PR TITLE
Create log filter to ignore record message if contains any set substring

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

File: docs/logs.md [hunk 1]
```diff
@@ -48,3 +48,33 @@ LOGGING = {
...
+
+#### Example
+
+```python
+LOGGING = {
+        'version': 1,
+        'filters': {
+            'ignore_if_contains': {
+                '()': 'django_toolkit.logs.filters.IgnoreIfContains',
+                'patterns': ['/ping', '/healthcheck'],
```
Comment (renanivo) [inline docs/logs.md]: What do you think about using `substring` instead of `pattern`? Because term pattern is usually used for regular expressions.

Comment (jairhenrique) [reply docs/logs.md]: I was thinking about it, but I couldn't find a better word.
I'll use substring, tks!

File: django_toolkit/logs/filters.py [hunk 2]
```diff
@@ -13,3 +13,21 @@ class AddHostName(logging.Filter):
...
+    def __init__(self, substrings=None):
+        self.substrings = substrings or []
+
+    def filter(self, record):
+        message = record.getMessage()
+
+        return not any([
+            substring in message
+            for substring in self.substrings
+        ])
```
Comment (drgarcia1986) [inline django_toolkit/logs/filters.py]: you can use gen-exp:
```python
return not any(
    substring in message
    for substring in self.substrings
)
```
You don't need to build a list with list comps before check bool values with `any`

Comment (jairhenrique) [reply django_toolkit/logs/filters.py]: 👍

**Review by renanivo — APPROVED** [review]

**Review by RomuloOliveira — APPROVED** [review]

**Review by drgarcia1986 — APPROVED** [review]

**Pull request merged**